webpack/ForemanTasks/Components/TasksTable/TasksTableConstants.js

```diff
@@ -19,5 +19,5 @@ export const UPDATE_CLICKED = 'UPDATE_CLICKED';
 
 export const TASKS_SEARCH_PROPS = {
   ...getControllerSearchProps('tasks'),
-  controller: 'foreman_tasks/tasks',
+  controller: 'foreman_tasks_tasks',
```

Comment:
@MariaAga do we count on this controller for other things except bookmarks? thanks

Reply:
only `SearchBar` uses this const, and it uses this for autocomplete and bookmarks

Reply:
cool, I saw no regression with the searchbar autocomplete,
this prop is for info and a unique key for the redux store, though the API calls are not affected.